spec/models/profile_spec.rb

```diff
@@ -52,7 +52,7 @@
       end
     end
 
-    describe '#contruct_full_name' do
+    describe '#construct_full_name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.